### ISSUE TITLE
vertica-mixed wrapper does not set job order correctly

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#438]
In GitLab by @etiennesky on Oct 10, 2019, 15:33

Hi @dbeltrankyl @mcastril I am trying to wrap 3 jobs in the vertical-mixed wrapper.

The experiment is a29g, AS version is  autosubmit/3.12.0b-foss-2015a-Python-2.7.9 and the config used is :

```
[wrapper]
TYPE = vertical-mixed
JOBS_IN_WRAPPER = SIM CMORVEG POST
```

in jobs.conf they are defined in the same order (SIM CMORVEG POST) and you can also see this order in the monitor plot at /esarchive/autosubmit/a29g/plot/a29g_20191010_1523.pdf 

![wrapped-jobs](https://earth.bsc.es/gitlab/es/autosubmit/uploads/f8eaeea62db6fa9f18ef9d247bd8cd7c/wrapped-jobs.png)

However, the wrapper job submitted contains the jobs in the wrong order (CMORVEG SIM POST):

in /esarchive/autosubmit/a29g/tmp/a29g_ASThread_15706329212889_96_3.cmd

```
# Defining scripts to be run                                                                                                                                                                                       
scripts = ['a29g_19790101_fc00_1_CMORVEG.cmd', 'a29g_19790101_fc00_1_SIM.cmd', 'a29g_19790101_fc00_1_POST.cmd']
```

It should at least respect the order defined in jobs.conf (as the docs say), but even better would be to respect the order given in JOBS_IN_WRAPPER

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Oct 10, 2019, 15:54

Hi @etiennesky ,

You're right, seems to be a bug there, although when fixed , it will likely be on 3.12.1b since a wrapper fix could potentially affect the version stability but I'll  consider the impact when done. 

FYI @wuruchi, since he will be looking to the wrapper code these weeks

**Comment by kinow** [comment]
In GitLab by @wuruchi on Oct 17, 2019, 14:58

Working on it. I mean, looking into it!

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Oct 25, 2019, 10:32

Hi, 

So there is a different issue then, not sure why it would pack the same jobs twice because it should look for the first job in a wrapper and from this job look the others, seems like it did the same process on another inner job when it shouldn't happen I'll check, 

@wuruchi did you checked on 3.12.0 version or 3.12.1b? the outcome of wrappers may differ.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Oct 25, 2019, 10:42

I see I checked it on 3.12.0b(cause @etiennesky is using it)  maybe it was partially solved on 3.12.1b with the outcome you saw.

**Comment by kinow** [comment]
In GitLab by @etiennesky on Oct 25, 2019, 10:51

I reported the problem with 3.12.0b
